drivers/spi/spi_cc13xx_cc26xx.c

```diff
@@ -9,14 +9,16 @@
 LOG_MODULE_REGISTER(spi_cc13xx_cc26xx);
 
 #include <drivers/spi.h>
-#include <sys_clock.h>
 
 #include <driverlib/prcm.h>
 #include <driverlib/ssi.h>
 #include <driverlib/ioc.h>
 
 #include "spi_context.h"
 
+/* CPU clock speed */
+#define CPU_CLOCK_HZ            ((uint32_t)48000000)
```

Comment:
We should have a DT define for this.  Look at boards/arm/sam_e70_xplained/sam_e70_xplained.dts and how it sets:

```&cpu0 {
        clock-frequency = <300000000>;
};
```

This should need something like in dts_fixup.h to use `DT_CPU_CLOCK_FREQUENCY`:

```
soc/arm/atmel_sam/same70/dts_fixup.h:#define DT_CPU_CLOCK_FREQUENCY     DT_ARM_CORTEX_M7_0_CLOCK_FREQUENCY
```

Reply:
Thanks @galak. I'll add this to dts.